index.bs

```diff
@@ -8,46 +8,190 @@ Repository: w3c/webrtc-dscp-exp
+      as defined in [[!RTCWEB-DATA]].</td>
+    </tr>
+    <tr>
+      <td><dfn enum-value><code>high</code></dfn></td>
+      <td>See [[!RTCWEB-TRANSPORT]], Sections 4.1 and 4.2. Corresponds to "extra
+      high" as defined in [[!RTCWEB-DATA]].</td>
+    </tr>
+  </tbody>
+</table>
 
-IDL specification {#idl}
-====
+Applications that use this API should be aware that often better
+overall user experience is obtained by lowering the priority of things
+that are not as important rather than raising the priority of the things
+that are.
 
+## Extension to RTCRtpEncodingParameters ## {#encoding-parameters}
 <pre class='idl'>
 partial dictionary RTCRtpEncodingParameters {
-    RTCPriorityType networkPriority;  // Note: No default
+  RTCPriorityType priority;
+  RTCPriorityType localPriority;
+  RTCPriorityType networkPriority;
 };
 </pre>
 
-networkPriority definition {#networkPriority}
-----
+<dl>
+  <dt><dfn dict-member for=RTCRtpEncodingParameters><code>priority</code></dfn> of type {{RTCPriorityType]], defaulting to
+    <code>"low"</code></dt>
+  <dd>
+    <p>
+      Indicates the priority of an {{RTCRtpSender}}, which influences the
+      bandwidth allocation among {{RTCRtpSender}} objects. It is specified
+      in [[!RTCWEB-TRANSPORT]], Section 4. The user agent is free to sub-allocate bandwidth
+      between the encodings of an {{RTCRtpSender}}.
+    </p>
+  </dd>
+</dl>
+<div link-for-hint=RTCRtpEncodingParameters>
+<dfn dict-member for=RTCRtpEncodingParameters>localPriority</dfn> has the same
+effect as {{priority}}, except that it only affects the bandwidth allocation
+decisions for the generated packets, as described in [[RTCWEB-TRANSPORT]]
+section 4.1.
+
+If {{localPriority}} is unset, the DSCP markings of the generated
+packets are controlled by the {{priority}} member.
 
-<dfn for='RTCRtpEncodingParameters'>networkPriority</dfn> has the same
+
+<dfn dict-member for=RTCRtpEncodingParameters>networkPriority</dfn> has the same
 effect as {{priority}}, except that it only affects the DSCP markings of
 the generated packets, as described in [[RTCWEB-TRANSPORT]] section 4.2.
 
 If {{networkPriority}} is unset, the DSCP markings of the generated
 packets are controlled by the {{priority}} member.
+</div>
+
+
+# Extensions for RTCDataChannel # {#datachannel}
+
+<pre class='idl'>
+partial interface RTCDataChannel {
+  readonly attribute RTCPriorityType priority;
+};
+
+partial dictionary RTCDataChannelInit {
+  RTCPriorityType priority;
+};
+
+</pre>
+## New RTCDataChannel attribute
+<dl>
+  <dt><dfn for=RTCDataChannel attribute>priority</dfn> of type {{RTCPriorityType}}, readonly</dt>
+  <dd>
+    <p>The <code>priority</code> attribute returns the priority for
+      this {{RTCDataChannel}}. The priority is assigned
+      by the user agent at channel creation time. On getting, the
+      attribute MUST return the value of the
+      [[\DataChannelPriority]] slot.</p>
```

Comment:
The`\` here is visible in the document, and formatting doesn't match uses elsewhere.